modules/hostap/src/supp_events.h

```diff
@@ -23,7 +23,9 @@ enum net_event_wpa_supp_cmd {
 	NET_EVENT_WPA_SUPP_CMD_IFACE_ADDED,
 	NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVING,
 	NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVED,
-	NET_EVENT_WIFI_CMD_MAX
+	NET_EVENT_WPA_SUPP_CMD_COUNT,
+	NET_EVENT_WPA_SUPP_CMD_MAX,
+	NET_EVENT_WPA_SUPP_CMD_INT_EVENT
```

Comment:
This has to be before the MAX and you can remove `NET_EVENT_WPA_SUPP_CMD_COUNT`.